static-crawling/static-crawling_assignment.py

```
@@ -0,0 +1,176 @@
+    html = response.text
+    soup = BeautifulSoup(html, 'html.parser')
+    trees = soup.select('.ss_book_box')
+    for tree in trees:
+        try:
+            title = tree.select_one('.bo3')
+            title_text = title.text
+            title_link = title.attrs['href']
+            price = tree.select_one('.ss_p2')
+            review = tree.select_one('.star_score')
+            datas.append([title_text, title_link, price.text, review.text])
+        except: continue
```

Ruff (0.13.1)

Do not use bare `except`

(E722)